src/main/java/com/leinardi/pycharm/mypy/toolwindow/TogglableTreeNode.java

```diff
@@ -44,9 +44,10 @@ public void setVisible(final boolean visible) {
         this.visible = visible;
     }
 
-    @SuppressWarnings("unchecked")
     List<TogglableTreeNode> getAllChildren() {
-        return Collections.unmodifiableList(children);
+        return children.stream()
+                .map(child -> (TogglableTreeNode) child)
+                .collect(Collectors.toList());
```

Comment:
This was failing some checks, stole this fix from https://github.com/jshiell/checkstyle-idea/blob/main/src/main/java/org/infernus/idea/checkstyle/toolwindow/TogglableTreeNode.java#L32-L36